src/scripts/main.js

```diff
@@ -1,3 +1,57 @@
 'use strict';
 
 // write code here
+const MAX_LENGTH = 9;
+const MAX_WIDTH = 9;
+const MIN_LENGTH = 2;
+const MIN_WIDTH = 2;
```

Comment:
You don't need to create 4 variables.
```suggestion
const MAX_SIZE = 9;
const MIN_SIZE = 2;
```

Reply:
Well, that`s true. But 4 variables make it more flexible, that is why I decided to do it).